tests/integration/test_api_gateway.py

```diff
@@ -353,3 +393,17 @@ def connect_api_gateway_to_http_with_lambda_proxy(self, gateway_name, target_uri
             resources=resources,
             stage_name=self.TEST_STAGE_NAME
         )
+
+    def create_lambda_for_sqs_event_source(self, fnName):
+        zip_file = testutil.create_lambda_archive(
```

Comment:
Great idea to use a util function here - can we also re-use this function in the two other places where we're creating Lambda functions in this file?